meeting_notes/20180517_planning-meeting-notes.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Meeting Notes - May 17, 2018
+location: HackLab.TO
+attendees: 18
+date: 2018-05-17
+startTime: 18:30
+endTime: 21:00
+---
+
+## Attendees
+
+dasanchez  
+Nicolas from AlterMundi/LibreMesh  
```

Comment:
`Nicolas Pace from Alter Mundi / Libre Router` (because there are two Nicolas from that group)